lib/config.js

```diff
@@ -45,6 +45,14 @@ const paramsSchema = Joi.object({
       .maxArity(1)
       .optional()
       .default(() => defaultSessionIdGenerator),
+    verificationCookie: Joi.object({
```

Comment:
Since the transaction cookie isn't strictly part of the session, can we move this config up to the root.

Also, I'm going to need to think about if other properties need to be configurable.

Also, can we call it `transactionCookie`

Reply:
Renamed it. I kept it in object format to allow for more options, but I'm unsure if there's currently a use case for modifying the other cookie settings.